sal_interfaces/MTVMS/MTVMS_Telemetry.xml

```diff
@@ -14,161 +14,276 @@
     <item>
       <EFDB_Name>sensor1XAcceleration</EFDB_Name>
```

Comment:
DM has a nice general naming convention (https://developer.lsst.io/cpp/style.html#general-naming-conventions) we should try to adopt as much as possible.

One of the guidelines state:

> The fundamental quantity being described should appear first in the name, with modifiers concatenated afterward. 

I would suggest changing from `sensor1Xacceleration` to `accelerationXSensor1` (preferred) or `xAccelerationSensor1`. Same throughout the other topics attributes.

Reply:
done, renamed to accelerationXSensor1 etc.